pages/index.vue

```diff
@@ -116,11 +107,11 @@
                     </div>
                 </div>
                 <div class="templates">
-                    <h2>Templates</h2>
-                    <p class="font-italic">Start by choosing a template</p>
+                    <h2>Layout</h2>
+                    <p class="font-italic">Do you need only a panel or a strip full of panels?</p>
```

Comment:
Suggestion for new text:
"Start by choosing the layout of your comic. You can either create one panel in different sizes or create a comic strip with multiple panels."